models/contact/src/index.ts

```diff
@@ -162,6 +174,33 @@ export function createModel (builder: Builder): void {
     contact.app.Contacts
   )
 
+  const contactLookup: Lookup<Contact> = {
```

Comment:
Table generate lookup, you don't need this in base cases

Reply:
Will remove on next PR.